api.bs

```diff
@@ -2344,6 +2344,53 @@ for a number of reasons:
     Without allocating [=privacy budget=] for new data,
     sites could exhaust their budget forever.
 
+### Formal Analysis of Privacy Properties and Their Limitations ### {#formal-analysis}
+ The formal privacy analysis is based on two papers. The first [[PPA-DP]] establishes the theory
+for on-device Individual DP accounting. The second [[PPA-DP-2]] provides formal analysis of
+the mathematical privacy guarantees afforded by *per-site budgets* and by *safety limits*
+(Section 3 is about per-site guarantees and Section 3.4 specifies the assumptions under
+which they hold). Per-site budgets include [=site=] in the [=privacy unit=], whereas safety
+limits exclude it thereby enforcing a global individual DP guarantee.  In Attribution Level 1
+it is conversion sites that have per-site budgets tracked.
+
+
+The analysis shows that *per-site individual DP guarantees* hold under a restricted system
+model that makes two assumptions, which may not always be satisfied in practice:
+
+1.  *No cross-site adaptivity in data generation.* A site's queryable data stream (impressions
+    and conversions) must be generated independently of past DP query results from other sites.
+1.  *No leakage through cross-site shared limits.* Queries from one site must not affect which
+    reports are emitted to others.
+
+Assumption 1 is necessary because the system involves multiple sites that could interact
+with the same user over time and change the ads they show to the user, or impact the
+conversions the user has, based on each other’s DP measurements. For example, if one advertiser
+learns, from DP measurements, to make an ad more effective, a user may convert on their site
```

Comment:
```suggestion
learns generally-applicable information that helps them make their ads more effective,
that will make it more likely that their ads are attributed for conversions,
as opposed to a competitor.
```

Here, "DP measurements" refers to `measureConversion()` as well.  Watch out for that.

Reply:
This a case where we are talking about aggregate results from many devices that you get back from the aggregation service. 

I feel we need some sort of term for this in the spec as for what to call the final aggregate DP results that go back to the advertiser.   Maybe "query" is too general but something like  "DP attribution results" or "DP measurements" should be clear and maybe we need to define that somewhere in the intro.